src/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title><%= htmlWebpackPlugin.options.title %></title>
+</head>
+<body>
+    <h1>Leaderboard</h1>
+    <main>
+        <div class="scores">
+            <div class="scores-header">
+                <h2>Recent Scores</h2>
+                <button id="refresh" type="button">Refresh</button>
+            </div>
+            <div class="recent-scores"></div>
+        </div>
+        <div class="add-score">
+            <h2>Add your score</h2>
+            <form action="#">
+                <input required type="text" name="name" id="userName" placeholder="Your Name">
+                <input required type="number" name="name" id="userScore" placeholder="Your Score">
+                <button id="submitScore" type="submit">Submit</button>

```

Comment:
- [ ] As the last code reviewer said you should fix the error when clicking on the submit button.

![image](https://user-images.githubusercontent.com/38635158/168419334-4fbcdc13-d11f-4bf1-a4f9-623255fdcc07.png)

Reply:
Hi @ahmetbozaci  
Thanks for the review, but I can't see any of those errors these errors in my console.
![image](https://user-images.githubusercontent.com/91727952/168419836-be110437-6ffd-4a1e-a112-5d7c3ddc44c4.png)

Reply:
If you have any suggestions, I would appreciate your help.

Reply:
@Christianib003 Answered in Slack.